src/Features/VisualBasic/Portable/UseAutoProperty/VisualBasicUseAutoPropertyAnalyzer.vb

```diff
@@ -172,7 +172,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UseAutoProperty
             For Each ref In containingType.DeclaringSyntaxReferences
                 Dim containingNode = ref.GetSyntax(cancellationToken)?.Parent
                 If containingNode IsNot Nothing Then
+#Disable Warning RS1030 ' Do not invoke Compilation.GetSemanticModel() method within a diagnostic analyzer
```

Comment:
Why is this being suppressed? You already added a new property `SemanticModel` to the base type for this analyzer in this PR, correct?

Reply:
While it's possible that `ref` is in the same tree, it's not guaranteed. Partial types can be split across multiple files which do not all use the same semantic model.

Reply:
Sure, but the common case should be all field and property definitions are in the same file. We should check for this case only create a new semantic model for uncommon path.

Reply:
I still couldn't remove the suppression, but I updated it to use the known semantic model when valid.